.github/workflows/kind.yml

```diff
@@ -49,6 +49,27 @@ jobs:
         path: antrea-ubuntu.tar
         retention-days: 1 # minimum value, in case artifact deletion by 'artifact-cleanup' job fails
 
+  build-antrea-ubi-image:
+    name: Build Antrea UBI image to be used for Kind e2e tests
+    needs: check-changes
+    if: ${{ needs.check-changes.outputs.has_changes == 'yes' }}
+    runs-on: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        show-progress: false
+    - name: Build Antrea UBI Docker image
+      run: |
+        ./hack/build-antrea-linux-all.sh --pull --distro ubi
+    - name: Save Antrea UBI image to tarball
+      run:  docker save -o antrea-ubi.tar antrea/antrea-ubi:latest
+    - name: Upload Antrea UBI image for subsequent jobs
+      uses: actions/upload-artifact@v3
+      with:
+        name: antrea-ubi
+        path: antrea-ubi.tar
+        retention-days: 1 # minimum value, in case artifact deletion by 'artifact-cleanup' job fails
```

Comment:
you didn't update `artifact-cleanup` actually

Reply:
thanks. reverted the change to kind.yml.